docs/source/schema-stitching.md

```diff
@@ -218,7 +218,7 @@ const transformedChirpSchema = transformSchema(chirpSchema, [
     (operation: string, rootField: string) => rootField !== 'chirpsByAuthorId'
   ),
   new RenameTypes((name: string) => `Chirp_${name}`),
-  new RenameRootFields((name: string) => `Chirp_${name}`),
+  new RenameRootFields((operation: string, name: string) => `Chirp_${name}`),
```

Comment:
The test here: https://github.com/chentsulin/graphql-tools/blob/052e1d2e825c63591e137e6a7760697da1def1f1/src/test/testAlternateMergeSchemas.ts#L69 probably needs updating too then